DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/RestaurantView.kt

```diff
@@ -104,7 +104,7 @@ private fun RestaurantActionButton() {
 }
 
 @Composable
-fun RestaurantView(viewWidth: Int, selectedIndex: Int, updateSelectedIndex: (Int) -> Unit, isDualScreen: Boolean) {
+fun RestaurantView(viewWidth: Int, selectedIndex: Int, updateSelectedIndex: (Int) -> Unit) {
```

Comment:
I think one of the DualView UI tests needs to be updated after this change

![image](https://user-images.githubusercontent.com/33138268/166706054-f619c812-56ba-404a-9dfb-b4dd2019ac8d.png)

Reply:
Good catch